src/transformers/models/trillsson_efficientnet/feature_extraction_trillsson_efficientnet.py

```diff
@@ -0,0 +1,513 @@
+        do_normalize=True,
+        return_attention_mask=False,
+        num_mel_bins=80,
+        log_floor=1e-12,
+        log_additive_offset=0.001,
+        window_length_secs=0.025,
+        hop_length_secs=0.010,
+        f_max=7500.0,
+        f_min=125.0,
+        mel_high_frequency_q=1127.0,
+        mel_break_frequency_hertz=700.0,
+        fft_length=None,
+        **kwargs
+    ):
+        super().__init__(feature_size=feature_size, sampling_rate=sampling_rate, padding_value=padding_value, **kwargs)
+        self.return_attention_mask = return_attention_mask
+        self.do_normalize = do_normalize
+        self.num_mel_bins = num_mel_bins
+        self.log_floor = log_floor
+        self.log_additive_offset = log_additive_offset
+        self.window_length_secs = window_length_secs
+        self.hop_length_secs = hop_length_secs
+        self.f_max = f_max
+        self.f_min = f_min
+        self.mel_high_frequency_q = mel_high_frequency_q
+        self.mel_break_frequency_hertz = mel_break_frequency_hertz
+        self.fft_length = fft_length
+
+    @classmethod
+    def pad_symmetric(cls, raw_speech: np.ndarray, padding: int):
+        """Symmetric pad a 2D Tensor"""
+        if raw_speech.ndim != 2:
+            raise ValueError(f"Raw speech must be rank 2 got {raw_speech.shape}")
+        raw_speech_len = raw_speech.shape[1]
+        if padding > raw_speech_len:
+            pad_raw_speech = cls.repeat_n_times_with_extra(raw_speech, padding, raw_speech_len)
+        else:
+            pad_raw_speech = np.pad(raw_speech, [(0, 0), (0, padding)], mode="symmetric")
+        return pad_raw_speech
+
+    @classmethod
+    def pad_waveform(cls, raw_speech: np.ndarray, padding: int, mode: str, padding_value: float):
+        """Pad waveform"""
+        if raw_speech.ndim != 2:
+            raise ValueError(f"Raw speech must be rank 2 got {raw_speech.shape}")
+        if mode == "symmetric":
+            return cls.pad_symmetric(raw_speech, padding)
+        else:
+            pad_raw_speech = np.pad(raw_speech, [(0, 0), (0, padding)], mode=mode, constant_values=padding_value)
+            return pad_raw_speech
+
+    @staticmethod
+    def repeat_n_times_with_extra(raw_speech: np.ndarray, padding: int, raw_speech_len: int):
+        """Pad symmetric longer than the original raw speech"""
+        if raw_speech.ndim != 2:
+            raise ValueError(f"Raw speech must be rank 2 got {raw_speech.shape}")
+        num_copies = np.floor_divide(padding, raw_speech_len)
+        r = np.fliplr(raw_speech)
+        f = np.concatenate((r, raw_speech), axis=1)
+        copies = np.tile(f, [1, np.floor_divide(num_copies, 2)])
+        if num_copies % 2 != 0:
+            copies = np.concatenate((copies, r), axis=1)
+        pre_pad = np.concatenate([raw_speech, copies], axis=1)
+        extra = padding % raw_speech_len
+        pad_raw_speech = np.pad(pre_pad, [(0, 0), (0, extra)], mode="symmetric")
+        return pad_raw_speech
```

Comment:
Same here, I think this is also supported.

Reply:
Same with above